src/__integrations__/pm.ts

```diff
@@ -122,7 +122,7 @@ walletRouter.get('/pp-restapi/v3/payments/:id/actions/check', function (req, res
   }
 });
 
-walletRouter.post('/pp-restapi/v3/wallet', function (req, res) {
+walletRouter.post('/pp-restapi/v4/wallet', function (req, res) {
```

Comment:
since we have decided to work on a local copy of api-spec , wouldn't it be appropriate to put the version in `basePath` of `api.yml` file?

Reply:
Yes, I'l change as you suggest